### PR TITLE
Milestone 01: Feature

### PR DESCRIPTION
**Pull request by mavericksbalitaan**
## Feature

Link: [hello-react-front-end](https://github.com/mavericks-db/hello-react-front-end)

**In this PR, I implement the following:**

- [x] Create a new Rails API app called 'hello-rails-back-end'.
- [x] Initialize your project with Git.
- [x] Make sure that your project has a Postgres database set up.
- [x] Create a table for storing your messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route)
---
- [x] In the README file for the Rails API, add a link to your React app.
- [x] When submitting for a review - use a link to the pull request in your Rails API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,108 @@
...
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Hello-Rails-Back-End
 
-Things you may want to cover:
+> This is an educational project to set up a "Hello world" project as two separated apps.
 
-* Ruby version
+Front-End: [Hello-Rails-Front-End](https://github.com/mavericks-db/hello-react-front-end)
```
Comment (ShoiraTa) [inline README.md:7]: - [ ] Please check the code on the front end, I am getting an infinite loop:

https://user-images.githubusercontent.com/77038610/189905818-7b431460-c75f-40b5-853a-44aacb865de5.mp4

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @mavericks-db 

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mavericksbalitaan) [reply README.md:7]: Resolved in the front-end component.

**Review by Munsa1 — APPROVED** [review]
Hi @mavericks-db, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**